extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java

```diff
@@ -630,6 +630,10 @@ private Properties getSchedulerConfigurationProperties(QuartzSupport quartzSuppo
         putExtensionConfigurationProperties(props, StdSchedulerFactory.PROP_TRIGGER_LISTENER_PREFIX,
                 buildTimeConfig.triggerListeners);
 
+        // As the very last step, pass all declared unsupported properties directly to Quartz
+        // Note that we are only passing in those that weren't already configured by user or Quarkus itself
+        quartzSupport.getRuntimeConfig().unsupportedProperties.forEach((key, value) -> props.putIfAbsent(key, value));
```

Comment:
The rule of trying to avoid lambdas in runtime code still applies, especially when they don't bring much :).

Reply:
Fair enough, changed to a for-each loop :)